changelog_entry.yaml

```
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    fixed:
+    - Update values for NJ TANF maximum benefit
```

It wasn't there previously to update right?
```suggestion
    - NJ TANF maximum benefit.
```